rhc.gemspec

```diff
@@ -30,7 +30,6 @@ Gem::Specification.new do |s|
     sep
   ].join("\n")
 
-  s.add_dependency              'net-ssh',      '>= 2.0.11', '<= 2.9.2'
   s.add_dependency              'net-scp',      '>= 1.1.2'
   s.add_dependency              'net-ssh-multi','>= 1.2.0'
   s.add_dependency              'archive-tar-minitar'
```

Comment:
Okay, I'm fine with this then. We just need to be aware that it is possible for the unlocked gems to cause issues in the future with updates.